elliottlib/util.py

```diff
@@ -237,10 +237,16 @@ def parallel_results_with_progress(inputs, func):
 
 
 def override_product_version(pv, branch):
-    """ if user specify -b branch will override product_version of erratatool.yml"""
+    """ Override product_version with given brew tags
```

Comment:
again i could use some pointers as to _why_ ... how is this useful?

Also I realize most of this logic is from previous iterations, but I'm really starting to feel this should just be a plain lookup from values in erratatool.yml; how would a product that's not OCP use this? Not saying this has to happen now, but we should probably have a story for it...

Reply:
This looks like a hack to mangle the RHEL-7 based Product Version defined in `erratatool.yml` to corresponding RHEL 8 based Product Version. For example, given product version `RHEL-7-OSE-4.1` and branch `rhaos-4.1-rhel-8-candidate`, this function will change the product version to `OSE-4.1-RHEL-8`.
I think Shiyang hasn't been finished this PR with the mapping of production version and branch  we talked on Thursday.